app/src/main/java/ro/code4/monitorizarevot/repositories/Repository.kt

```diff
@@ -125,15 +135,15 @@ class Repository : KoinComponent {
     fun getForms(): Observable<Unit> {
 
         val observableDb = db.formDetailsDao().getAllForms().toObservable()
-
         val observableApi = apiInterface.getForms()
 
+        // todo fix this as it does not work as expected. nulls are not treated as return values.
+        //  instead it throws a NullPointerException.
```

Comment:
RxJava2 doesn't allow null as a return value, it's well-known.

Reply:
Well, that is the comment here for. However, nonetheless the nulls were somehow added to the code and it does not work as expected. Thus, the comment you see.